.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           images: goniszewski/grimoire
           tags: |
             type=raw,value=latest,enable=${{ !github.event.release.prerelease }}
-            type=raw,value=preview,enable=${{ !github.event.release.prerelease }}
+            type=raw,value=preview,enable=${{ github.event.release.prerelease }}
```

Comment:
These are not my changes not sure why they are here

Reply:
Oh, I did change the target branch to `develop`, as it needs to be merged there first. Sorry for the confusion!